static/app/components/replays/replayController.tsx

```diff
@@ -50,6 +57,15 @@ function ReplayPlayPauseBar({isCompact}: {isCompact: boolean}) {
     togglePlayPause,
   } = useReplayContext();
 
+  const handlePlayToggle = () => {
+    trackAdvancedAnalyticsEvent('replay.play-pause', {
+      organization,
+      user_email: config.user.email,
+      play: !isPlaying,
+    });
```

Comment:
this could be better off if it were inside of ReplayContext. Then if we had other ways to trigger play/pause (like a keyboard shortcut, or if we wanted to log that someone got to the end of the replay) it'd be centralized

Also, it'll help keep the react components lean and focused on rendering stuff. The callbacks are already coming from the hook/context, so lets keep stuffing the code up there.